roles/validate/tasks/sub_main.yml

```diff
@@ -136,14 +136,10 @@
 - name: Read Run Map From Previous Run
   cisco.nac_dc_vxlan.common.read_run_map:
     model_data: "{{ MD_Extended }}"
+    play_tags: "{{ ansible_run_tags }}"
   register: run_map_read_result
   delegate_to: localhost
 
-- name: Debug Run Map Read Result
```

Comment:
I removed this because I display a warning in the action plugin if the diff_run feature is disabled for any reason